<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1231</width>
     <height>761</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(211, 215, 207);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_2">
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>280</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>340</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>340</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>220</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_pw">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>280</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_id">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>280</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>340</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>비밀번호확인</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_pw2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>340</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_idcheck">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>220</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>중복확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signup">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>400</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>회원가입</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_signUp">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>180</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>551</x>
       <y>700</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>440</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>매매현황</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>390</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>490</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>20</y>
       <width>421</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_4">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>490</y>
       <width>411</width>
       <height>260</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <column>
      <property name="text">
       <string>회사 명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>보유 주식</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="sell">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>590</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>매도</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>390</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>전송</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>490</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buy">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>590</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>매수</string>
     </property>
    </widget>
    <widget class="QLabel" name="time">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>440</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>261</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>회사 명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재 주가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>잔여 주식 수량</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>490</y>
       <width>261</width>
       <height>261</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>회사 명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재 주가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>잔여 주식 수량</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>771</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="graph"/>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>660</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
